## 82% of end-end tests have passed
```
Total executed tests: 51
Failed tests: 9
Expected to fail tests: 0
Passed tests: 42
Not executed tests: 1
```
```
IDs of not executed tests: 727232 
```
```
IDs of failed tests: 727230,703496,702783,702782,702784,727229,703391,739307,703297 
```
<h3>Not executed tests (1)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=fix/prevent_credential_leak_from_log&PR_ID=20986&APK_NAME=20986.apk&TR_CASE_IDS=727232">Rerun not executed tests</a></li></details><h3>Failed tests (9)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=fix/prevent_credential_leak_from_log&PR_ID=20986&APK_NAME=20986.apk&TR_CASE_IDS=727230,703496,702783,702782,702784,727229,703391,739307,703297">Rerun failed tests</a></li><br/><h4>Class TestDeepLinksOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3637026">test_deep_links_communities</a>, id: 739307 </b></td></tr><tr><td><p><blockquote><div>Device 1: ## Creating closed community </div><div>Device 1: Find `Button` by `accessibility id`: `new-communities-button` </div></blockquote></p>

```
critical/test_deep_and_universal_links.py:81: in test_deep_links_communities
    self.home.create_community(community_type="closed")
../views/home_view.py:492: in create_community
    self.plus_community_button.click()
../views/base_element.py:90: in click
    element = self.find_element()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: Button by accessibility id: `new-communities-button` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d28e3c5a93a4417d9dbfc63134b64ffd?auth=7e31976eac7d3899477d5ccfa1602705#192">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3636983">test_1_1_chat_emoji_send_reply_and_open_link</a>, id: 702782 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `OpenInStatusButton` by `xpath`: `//*[@text="Open in Status"]` </div><div>Device 2: Tap on found: OpenInStatusButton </div></blockquote></p>

```
critical/chats/test_1_1_public_chats.py:179: in test_1_1_chat_emoji_send_reply_and_open_link
    self.errors.verify_no_errors()
base_test_case.py:191: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Reply for 'Test with link: https://status.im/ here should be nothing unusual.' not present in message received in public chat
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/483763d1c0414ec39b51c218cda05c65?auth=180947357e3ac45ef02b7f01f06cb2ff#148">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/28a59f770cdb4ccfb917370341384080?auth=cae07284f2a4999d655b9221aea7ae18#118">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3637015">test_1_1_chat_send_image_save_and_share</a>, id: 703391 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `Button` by `accessibility id`: `image-0` </div><div>Device 2: Click system back button </div></blockquote></p>

```
critical/chats/test_1_1_public_chats.py:467: in test_1_1_chat_send_image_save_and_share
    self.errors.verify_no_errors()
base_test_case.py:191: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Message about saving a photo is not shown for sender.
E    Message about saving a photo is not shown for receiver.
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/483763d1c0414ec39b51c218cda05c65?auth=180947357e3ac45ef02b7f01f06cb2ff#489">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/28a59f770cdb4ccfb917370341384080?auth=cae07284f2a4999d655b9221aea7ae18#339">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3637013">test_group_chat_send_image_save_and_share</a>, id: 703297 </b></td></tr><tr><td><p><blockquote><div>Device 3: Find `ChatsTab` by `accessibility id`: `chats-stack-tab` </div><div>Device 3: Tap on found: ChatsTab </div></blockquote></p>

```
critical/chats/test_group_chat.py:297: in test_group_chat_send_image_save_and_share
    self.errors.verify_no_errors()
base_test_case.py:191: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Message about saving a photo is not shown for member_2.
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/32a0e42dcc98475fb18fd2a9b1283dd9?auth=ee7cd320bd4ca2a5e219e98360e9b807#279">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6e9dd401d0d5473c80b4d5eb229f54ea?auth=97bfa6e79cdacf5029c67d6c2041b0ef#169">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5ae1aef6cae342c7bba7c86a2bc633c4?auth=3bd751946251a8e0f1de020021b3639a#140">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3637023">test_wallet_send_asset_from_drawer</a>, id: 727230 </b></td></tr><tr><td>

```
Test setup failed: critical/test_wallet.py:29: in prepare_devices
    self.loop.run_until_complete(
/usr/lib/python3.10/asyncio/base_events.py:649: in run_until_complete
    return future.result()
__init__.py:52: in run_in_parallel
    returns.append(await k)
/usr/lib/python3.10/concurrent/futures/thread.py:58: in run
    result = self.fn(*self.args, **self.kwargs)
../views/sign_in_view.py:297: in recover_access
    self.start_button.click()
../views/base_element.py:90: in click
    element = self.find_element()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: Button by accessibility id: `welcome-button` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3637022">test_wallet_send_eth</a>, id: 727229 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Button` by `accessibility id`: `enable-notifications-later-button` </div><div>Device 1: Find `Button` by `accessibility id`: `welcome-button` </div></blockquote></p>

```
Test setup failed: critical/test_wallet.py:29: in prepare_devices
    self.loop.run_until_complete(
/usr/lib/python3.10/asyncio/base_events.py:649: in run_until_complete
    return future.result()
__init__.py:52: in run_in_parallel
    returns.append(await k)
/usr/lib/python3.10/concurrent/futures/thread.py:58: in run
    result = self.fn(*self.args, **self.kwargs)
../views/sign_in_view.py:297: in recover_access
    self.start_button.click()
../views/base_element.py:90: in click
    element = self.find_element()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: Button by accessibility id: `welcome-button` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUiTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3637017">test_1_1_chat_mute_chat</a>, id: 703496 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_1_1_public_chats.py:545: in prepare_devices
    self.loop.run_until_complete(run_in_parallel(((self.device_1.create_user, {'enable_notifications': True,
/usr/lib/python3.10/asyncio/base_events.py:649: in run_until_complete
    return future.result()
__init__.py:52: in run_in_parallel
    returns.append(await k)
/usr/lib/python3.10/concurrent/futures/thread.py:58: in run
    result = self.fn(*self.args, **self.kwargs)
../views/sign_in_view.py:269: in create_user
    self.start_button.click()
../views/base_element.py:90: in click
    element = self.find_element()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: Button by accessibility id: `welcome-button` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3636984">test_1_1_chat_is_shown_message_sent_delivered_from_offline</a>, id: 702783 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `Button` by `accessibility id`: `welcome-button` </div><div>Device 1: Find `Button` by `accessibility id`: `welcome-button` </div></blockquote></p>

```
Test setup failed: critical/chats/test_1_1_public_chats.py:545: in prepare_devices
    self.loop.run_until_complete(run_in_parallel(((self.device_1.create_user, {'enable_notifications': True,
/usr/lib/python3.10/asyncio/base_events.py:649: in run_until_complete
    return future.result()
__init__.py:52: in run_in_parallel
    returns.append(await k)
/usr/lib/python3.10/concurrent/futures/thread.py:58: in run
    result = self.fn(*self.args, **self.kwargs)
../views/sign_in_view.py:269: in create_user
    self.start_button.click()
../views/base_element.py:90: in click
    element = self.find_element()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: Button by accessibility id: `welcome-button` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d3aae6f06ae94ef3b69dff080492bb93?auth=c2fb88fe65ed2e2313a6165bd324c371">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8bb390ccc94d4ff9bc8dcf426ddb4615?auth=69e2737a80cc1bbc124a945fbeecd145">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3636985">test_1_1_chat_delete_via_long_press_relogin</a>, id: 702784 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_1_1_public_chats.py:545: in prepare_devices
    self.loop.run_until_complete(run_in_parallel(((self.device_1.create_user, {'enable_notifications': True,
/usr/lib/python3.10/asyncio/base_events.py:649: in run_until_complete
    return future.result()
__init__.py:52: in run_in_parallel
    returns.append(await k)
/usr/lib/python3.10/concurrent/futures/thread.py:58: in run
    result = self.fn(*self.args, **self.kwargs)
../views/sign_in_view.py:269: in create_user
    self.start_button.click()
../views/base_element.py:90: in click
    element = self.find_element()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: Button by accessibility id: `welcome-button` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr></tbody></table></details><h3>Passed tests (42)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3637001">test_1_1_chat_edit_message</a>, id: 702855 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/483763d1c0414ec39b51c218cda05c65?auth=180947357e3ac45ef02b7f01f06cb2ff#458">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/28a59f770cdb4ccfb917370341384080?auth=cae07284f2a4999d655b9221aea7ae18#289">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3636975">test_1_1_chat_message_reaction</a>, id: 702730 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/483763d1c0414ec39b51c218cda05c65?auth=180947357e3ac45ef02b7f01f06cb2ff#94">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/28a59f770cdb4ccfb917370341384080?auth=cae07284f2a4999d655b9221aea7ae18#83">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3636980">test_1_1_chat_non_latin_messages_stack_update_profile_photo</a>, id: 702745 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/483763d1c0414ec39b51c218cda05c65?auth=180947357e3ac45ef02b7f01f06cb2ff#351">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/28a59f770cdb4ccfb917370341384080?auth=cae07284f2a4999d655b9221aea7ae18#208">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3636976">test_1_1_chat_pin_messages</a>, id: 702731 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/483763d1c0414ec39b51c218cda05c65?auth=180947357e3ac45ef02b7f01f06cb2ff#213">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/28a59f770cdb4ccfb917370341384080?auth=cae07284f2a4999d655b9221aea7ae18#163">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3636978">test_1_1_chat_text_message_delete_push_disappear</a>, id: 702733 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/483763d1c0414ec39b51c218cda05c65?auth=180947357e3ac45ef02b7f01f06cb2ff#547">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/28a59f770cdb4ccfb917370341384080?auth=cae07284f2a4999d655b9221aea7ae18#396">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3636990">test_1_1_chat_push_emoji</a>, id: 702813 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/483763d1c0414ec39b51c218cda05c65?auth=180947357e3ac45ef02b7f01f06cb2ff#409">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/28a59f770cdb4ccfb917370341384080?auth=cae07284f2a4999d655b9221aea7ae18#258">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3636993">test_community_emoji_send_copy_paste_reply</a>, id: 702840 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/213cd05ed5e942169ee2d4717523adcc?auth=ee159d43f6a3e323c4cacc593aed687f#304">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/aed6baefce4a4489907ae0fa0c105817?auth=888b7e6d543f3daed8ac239bc3f0f921#282">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3637004">test_community_contact_block_unblock_offline</a>, id: 702894 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/213cd05ed5e942169ee2d4717523adcc?auth=ee159d43f6a3e323c4cacc593aed687f#423">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/aed6baefce4a4489907ae0fa0c105817?auth=888b7e6d543f3daed8ac239bc3f0f921#392">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3637009">test_community_mark_all_messages_as_read</a>, id: 703086 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/213cd05ed5e942169ee2d4717523adcc?auth=ee159d43f6a3e323c4cacc593aed687f#550">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/aed6baefce4a4489907ae0fa0c105817?auth=888b7e6d543f3daed8ac239bc3f0f921#471">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3636996">test_community_links_with_previews_github_youtube_twitter_gif_send_enable</a>, id: 702844 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/213cd05ed5e942169ee2d4717523adcc?auth=ee159d43f6a3e323c4cacc593aed687f#328">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/aed6baefce4a4489907ae0fa0c105817?auth=888b7e6d543f3daed8ac239bc3f0f921#301">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3636994">test_community_unread_messages_badge</a>, id: 702841 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/213cd05ed5e942169ee2d4717523adcc?auth=ee159d43f6a3e323c4cacc593aed687f#393">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/aed6baefce4a4489907ae0fa0c105817?auth=888b7e6d543f3daed8ac239bc3f0f921#362">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3636992">test_community_message_delete</a>, id: 702839 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/213cd05ed5e942169ee2d4717523adcc?auth=ee159d43f6a3e323c4cacc593aed687f#199">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/aed6baefce4a4489907ae0fa0c105817?auth=888b7e6d543f3daed8ac239bc3f0f921#134">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3636991">test_community_message_send_check_timestamps_sender_username</a>, id: 702838 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/213cd05ed5e942169ee2d4717523adcc?auth=ee159d43f6a3e323c4cacc593aed687f#157">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/aed6baefce4a4489907ae0fa0c105817?auth=888b7e6d543f3daed8ac239bc3f0f921#105">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3637021">test_community_edit_delete_message_when_offline</a>, id: 704615 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/213cd05ed5e942169ee2d4717523adcc?auth=ee159d43f6a3e323c4cacc593aed687f#585">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/aed6baefce4a4489907ae0fa0c105817?auth=888b7e6d543f3daed8ac239bc3f0f921#497">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>9. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3637002">test_community_one_image_send_reply</a>, id: 702859 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/213cd05ed5e942169ee2d4717523adcc?auth=ee159d43f6a3e323c4cacc593aed687f#256">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/aed6baefce4a4489907ae0fa0c105817?auth=888b7e6d543f3daed8ac239bc3f0f921#226">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>10. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3636995">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/213cd05ed5e942169ee2d4717523adcc?auth=ee159d43f6a3e323c4cacc593aed687f#175">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/aed6baefce4a4489907ae0fa0c105817?auth=888b7e6d543f3daed8ac239bc3f0f921#122">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>11. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3637011">test_community_several_images_send_reply</a>, id: 703194 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/213cd05ed5e942169ee2d4717523adcc?auth=ee159d43f6a3e323c4cacc593aed687f#228">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/aed6baefce4a4489907ae0fa0c105817?auth=888b7e6d543f3daed8ac239bc3f0f921#163">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3636997">test_community_leave</a>, id: 702845 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e5e4aeeae2f54f509ca47ead70da4b20?auth=fefbb44ac6381d3952b7f373a687022b#299">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e574df4894d14164bd6e0b52eb5272d2?auth=4d8f1cd08241ff37fc3f1486a91f9694#269">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3636986">test_community_mentions_push_notification</a>, id: 702786 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e5e4aeeae2f54f509ca47ead70da4b20?auth=fefbb44ac6381d3952b7f373a687022b#134">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e574df4894d14164bd6e0b52eb5272d2?auth=4d8f1cd08241ff37fc3f1486a91f9694#108">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3636989">test_community_markdown_support</a>, id: 702809 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e5e4aeeae2f54f509ca47ead70da4b20?auth=fefbb44ac6381d3952b7f373a687022b#183">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e574df4894d14164bd6e0b52eb5272d2?auth=4d8f1cd08241ff37fc3f1486a91f9694#167">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3637006">test_community_hashtag_links_to_community_channels</a>, id: 702948 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e5e4aeeae2f54f509ca47ead70da4b20?auth=fefbb44ac6381d3952b7f373a687022b#302">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e574df4894d14164bd6e0b52eb5272d2?auth=4d8f1cd08241ff37fc3f1486a91f9694#291">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3637019">test_community_join_when_node_owner_offline</a>, id: 703629 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e5e4aeeae2f54f509ca47ead70da4b20?auth=fefbb44ac6381d3952b7f373a687022b#477">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e574df4894d14164bd6e0b52eb5272d2?auth=4d8f1cd08241ff37fc3f1486a91f9694#454">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3637012">test_group_chat_reactions</a>, id: 703202 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/32a0e42dcc98475fb18fd2a9b1283dd9?auth=ee7cd320bd4ca2a5e219e98360e9b807#172">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6e9dd401d0d5473c80b4d5eb229f54ea?auth=97bfa6e79cdacf5029c67d6c2041b0ef#105">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5ae1aef6cae342c7bba7c86a2bc633c4?auth=3bd751946251a8e0f1de020021b3639a#85">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3636987">test_group_chat_join_send_text_messages_push</a>, id: 702807 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/32a0e42dcc98475fb18fd2a9b1283dd9?auth=ee7cd320bd4ca2a5e219e98360e9b807#155">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6e9dd401d0d5473c80b4d5eb229f54ea?auth=97bfa6e79cdacf5029c67d6c2041b0ef#77">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5ae1aef6cae342c7bba7c86a2bc633c4?auth=3bd751946251a8e0f1de020021b3639a#77">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3636988">test_group_chat_offline_pn</a>, id: 702808 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/32a0e42dcc98475fb18fd2a9b1283dd9?auth=ee7cd320bd4ca2a5e219e98360e9b807#319">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6e9dd401d0d5473c80b4d5eb229f54ea?auth=97bfa6e79cdacf5029c67d6c2041b0ef#204">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5ae1aef6cae342c7bba7c86a2bc633c4?auth=3bd751946251a8e0f1de020021b3639a#201">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3636977">test_group_chat_pin_messages</a>, id: 702732 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/32a0e42dcc98475fb18fd2a9b1283dd9?auth=ee7cd320bd4ca2a5e219e98360e9b807#353">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6e9dd401d0d5473c80b4d5eb229f54ea?auth=97bfa6e79cdacf5029c67d6c2041b0ef#231">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5ae1aef6cae342c7bba7c86a2bc633c4?auth=3bd751946251a8e0f1de020021b3639a#228">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3637016">test_group_chat_mute_chat</a>, id: 703495 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/32a0e42dcc98475fb18fd2a9b1283dd9?auth=ee7cd320bd4ca2a5e219e98360e9b807#473">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6e9dd401d0d5473c80b4d5eb229f54ea?auth=97bfa6e79cdacf5029c67d6c2041b0ef#274">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5ae1aef6cae342c7bba7c86a2bc633c4?auth=3bd751946251a8e0f1de020021b3639a#242">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3637005">test_activity_center_reply_read_unread_delete_filter_swipe</a>, id: 702947 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1840c4fd0f734d6caf14fe3b2b8abc5d?auth=e4e26ad7f956f4f5e5b23b591da846b6#127">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c58b57ff29ec40acaac7652265729a80?auth=3d5641eeba2879c85b17a560efbbe005#99">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePRTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3637008">test_activity_center_admin_notification_accept_swipe</a>, id: 702958 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9b28ce855c9141c9abecbe31d8aef349?auth=99b4838bcdd52ce0498a7d2c80bd6ad4#168">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b793a891777747fcb333e762f0f20022?auth=54fab43472bc35de7d08919fdfeef8b2#115">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3637007">test_activity_center_mentions</a>, id: 702957 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9b28ce855c9141c9abecbe31d8aef349?auth=99b4838bcdd52ce0498a7d2c80bd6ad4#127">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b793a891777747fcb333e762f0f20022?auth=54fab43472bc35de7d08919fdfeef8b2#99">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityCenterContactRequestMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3637000">test_activity_center_contact_request_accept_swipe_mark_all_as_read</a>, id: 702851 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f4526abff28341baa7caa2893e8f935f?auth=866cc4c3c621d203441b9106d086d67c#98">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e62e48d9e23e4c46b9a0ae0ec6c79b98?auth=53c96e1e70098e7be6ae5ecbb633bbdd#116">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3636999">test_activity_center_contact_request_decline</a>, id: 702850 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f4526abff28341baa7caa2893e8f935f?auth=866cc4c3c621d203441b9106d086d67c#55">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e62e48d9e23e4c46b9a0ae0ec6c79b98?auth=53c96e1e70098e7be6ae5ecbb633bbdd#54">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3636982">test_add_contact_field_validation</a>, id: 702777 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f4526abff28341baa7caa2893e8f935f?auth=866cc4c3c621d203441b9106d086d67c#133">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e62e48d9e23e4c46b9a0ae0ec6c79b98?auth=53c96e1e70098e7be6ae5ecbb633bbdd#200">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3637024">test_wallet_add_remove_regular_account</a>, id: 727231 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/52dd0a054918458191219c09cae4d057?auth=c42e629f1e3545e541433c1749ac43a9#37">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3636979">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/fefce13bece34bd59b253690d7c395b6?auth=2ee3065227648499d42f6e91d24ef7b3#75">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3636998">test_community_navigate_to_channel_when_relaunch</a>, id: 702846 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/fefce13bece34bd59b253690d7c395b6?auth=2ee3065227648499d42f6e91d24ef7b3#51">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3637010">test_restore_multiaccount_with_waku_backup_remove_switch</a>, id: 703133 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/fefce13bece34bd59b253690d7c395b6?auth=2ee3065227648499d42f6e91d24ef7b3#228">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3637003">test_community_undo_delete_message</a>, id: 702869 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/fefce13bece34bd59b253690d7c395b6?auth=2ee3065227648499d42f6e91d24ef7b3#104">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3637014">test_community_mute_community_and_channel</a>, id: 703382 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/fefce13bece34bd59b253690d7c395b6?auth=2ee3065227648499d42f6e91d24ef7b3#123">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3637018">test_community_discovery</a>, id: 703503 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/fefce13bece34bd59b253690d7c395b6?auth=2ee3065227648499d42f6e91d24ef7b3#352">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestDeepLinksOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3637020">test_links_open_universal_links_from_chat</a>, id: 704613 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d28e3c5a93a4417d9dbfc63134b64ffd?auth=7e31976eac7d3899477d5ccfa1602705#59">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3636981">test_links_deep_links_profile</a>, id: 702775 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d28e3c5a93a4417d9dbfc63134b64ffd?auth=7e31976eac7d3899477d5ccfa1602705#122">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

--- a/status-go-version.json
+++ b/status-go-version.json
@@ -3,7 +3,7 @@
     "_comment": "Instead use: scripts/update-status-go.sh <rev>",
     "owner": "status-im",
     "repo": "status-go",
-    "version": "v0.182.41",
-    "commit-sha1": "6581f75c637363317e4ca9b5571b2c6fd856148d",
-    "src-sha256": "006swly0h6fa061wk5g2szx1xl71gay47mn9c6qm1im601pxhgjs"
+    "version": "v0.182.42",
+    "commit-sha1": "a93c857238c3002ae64da1bdbebea13f3392da8b",
+    "src-sha256": "0rrny83rnirpm5pwp8br6nnglijb4whx1xl6nwkm5lzfnnichg59"
 }

